documents/service.go

```diff
@@ -190,10 +191,25 @@ func (s service) RequestDocumentSignature(ctx context.Context, model Model, coll
 
 	srvLog.Infof("document received %x with signing root %x", model.ID(), sr)
 
-	sig, err := acc.SignMsg(sr)
+	var transitionValidated bool
+	var transitionFlag int8
+	// If there is a previous version and we have successfully validated the transition then set the signature flag
+	if old != nil {
+		transitionValidated = true
+		transitionFlag = 1
```

Comment:
is there a case where we set the `transitionValidated=false` and `transitionFlag=1` and vice versa? If not, why define the same thing twice?

Reply:
You are right, it can be simplified.